soc/v2/nuvoton/numaker/Kconfig.soc

```diff
@@ -2,17 +2,12 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
-
 config SOC_FAMILY_NUMAKER
 	select PLATFORM_SPECIFIC_INIT
```

Comment:
I believe this line should be moved to the Zephyr specific `Kconfig`, not `Kconfig.soc`

Reply:
fixed